src/pat/contentbrowser/src/RecentlyUsed.svelte

```diff
@@ -30,7 +30,7 @@
             {#each items.reverse() as recentlyUsed}
                 <li>
                     <a
-                        href={recentlyUsed.getURL}
+                        href={recentlyUsed.getURL || "#"}
```

Comment:
different scope than the component registry work here, right?

Reply:
`RecentlyUsed` is currently not registered ... this is only a fix unrelated to this PR.